src/network/networkd-sysctl.c

```diff
@@ -40,10 +42,82 @@ static void manager_set_ip_forwarding(Manager *manager, int family) {
                                   enable_disable(t), af_to_ipv4_ipv6(family));
 }
 
+static void sysctl_overwrite_check(sd_event_source *source, const struct inotify_event *event, Hashmap *sysctls) {
+        _cleanup_free_ char *buf = NULL, *path = NULL;
+        const char *dir_path, *value;
+        size_t len;
+        int r;
+
+        assert(event);
+
+        r = sd_event_source_get_inotify_path(source, &dir_path);
+        if (r < 0)
+                return;
```

Comment:
so we usually prefer these days if errors are ignored by the caller, not by the callee when reasonable.

hence, I think we should just do `return r;` in this error path, and then in the caller just cast the return value to void, i.e. call as `(void) sysctl_overwrite_check(…);`.

Reply:
and also add `log_debug_errno(r, …` with some brief string.